backend/runners/load_representative_points.py

```diff
@@ -44,17 +65,23 @@ def _main(kwargs):
     with open(kwargs['filepath'], 'r') as f:
         json_features = geojson.load(f)['features']
 
-    for feature in json_features:
-        feature['properties']['zip_code'] = FAKE_ZIP_CODE
+    if kwargs['fake_state'] or FAKE_COUNTY:
```

Comment:
My preference is to force the values in this case -- if `fake_state is True`, any existing values in the data should be ignored. Otherwise maybe a single feature has a valid value that doesn't get overwritten, which could result in things being hard to debug down the line. ("I loaded this with `fake_state=='CA'`, why is there Oregon data?")